docs/dev/get_the_code_build_and_run.md

```diff
@@ -0,0 +1,53 @@
+# Get the code, build, and run
```

Comment:
It is better to introduce how to get the code, build, and test. how to run with tispark jar may suitable for user guide